### DISCUSSION TITLE
Is it possible to monitor PVC usage?

### DISCUSSION THREAD

**Discussion by hillbun** [prometheus-operator/kube-prometheus#1015] [General]
Is it possible to monitor PVC usage and alert?

**Comment by metalmatze** [comment]
Yes, that's exposed by the kubelet which is one of the targets scraped by the prometheus-k8s out-of-the-box.

Look for metrics like `kubelet_volume_stats_*`.

**Comment by hillbun** [comment]
@metalmatze 

Which dashboard I can view usage graph?

and which built-in alert rule?

**Reply by TheMeier** [reply]
KubePersistentVolumeFillingUp (or KubePersistentVolumeUsageCritical depending on Version)
KubePersistentVolumeErrors
From: https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/alerts/storage_alerts.libsonnet
